main.go

```diff
@@ -1,16 +1,21 @@
 package main
 
 import (
-	_ "embed"
+	"embed"
 
 	"github.com/defenseunicorns/zarf/src/cmd"
 	"github.com/defenseunicorns/zarf/src/config"
 )
 
+//go:embed all:build/ui/*
+var assets embed.FS
+
 //go:embed cosign.pub
-var cosignPublicKey string
+var cosignPublicKeyUI string
```

Comment:
I have no issue with this name change but I don't know if I understand why it was changed 🤷

Reply:
error from the native app stuff